concepts/05 Widgets/Sankey/05 Nodes and Links/10 Hover/10 Events.md

```diff
@@ -58,6 +58,75 @@ The **Sankey** widget raises individual events for nodes and links when their ho
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxSankey
+            @nodeHoverChanged="onNodeHoverChanged"
+            @linkHoverChanged="onLinkHoverChanged"
+        />
+    </template>
+
+    <script>
+    import DxSankey from 'devextreme-vue/sankey';
+
+    export default {
+        components: {
+            DxSankey
+        },
+        methods: {
+            onNodeHoverChanged (e) {
+                if(e.target.isHovered()) {
+                    // Commands to execute when the pointer enters a node
+                } else {
+                    // Commands to execute when the pointer leaves a node
+                }
+            },
+            onLinkHoverChanged (e) {
+                if(e.target.isHovered()) {
+                    // Commands to execute when the pointer enters a link
+                } else {
+                    // Commands to execute when the pointer leaves a link
+                }
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import Sankey from 'devextreme-react/sankey';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Sankey
+                    onNodeHoverChanged={onNodeHoverChanged}
```

Comment:
I see that onNode/LinkHoverChanged are declared inside the class. Shouldn't we write like {this.onNodeHoverChanged}?